nodejs/lancedb/table.ts

```diff
@@ -479,6 +482,18 @@ export class Table {
   async toArrow(): Promise<ArrowTable> {
     return await this.query().toArrow();
   }
+  /** List all the stats of a specified index
+   *
+   * @param {string} name The name of the index. The index name should always be `${column}_idx`
```

Comment:
>  The index name should always be `${column}_idx`

I don't know if that's something we need to enforce at all. In Lance, you can name it whatever you want. But I suppose LanceDB probably names it for you, based on the column name.

I'd maybe say instead where you can check what indices are currently on the table and what names they have. Can say the convention is `${column}_idx`.

Reply:
I did add a comment on `createIndex` that states that we don't yet support custom named indices.
```ts
   * @note We currently don't support custom named indexes,
   * The index name will always be `${column}_idx`
```

So i'll just remove that comment from `listIndices()`